x-pack/plugins/canvas/canvas_plugin_src/functions/common/timefilter.test.js

```diff
@@ -13,9 +13,10 @@ import { timefilter } from './timefilter';
 
 const errors = getFunctionErrors().timefilter;
 
-let clock = null;
+const fn = functionWrapper(timefilter);
```

Comment:
Are the changes here needed?

Reply:
No, right now it is ok to revert changes.

Reply:
Done.